### ISSUE TITLE
Port to Plone 5.2 and Python 3

### ISSUE THREAD

**Issue by jensens** [collective/collective.fingerpointing#97]
Port to 
- [x] Plone 5.2 
- [x] Python 3.6/3.7

keep track of state at https://github.com/orgs/collective/projects/1

**Comment by jensens** [comment]
When having *collective.fingerpointing* installed and running *plone-compile-resources* in Plone 5.2 on Python 2.7:

In ``logger.py`` the ``logging.handlers`` in line 39 can not be accessed and I get     

```
File "/home/jensens/workspace/pro/akifair/parts/instance/etc/site.zcml", line 15.2-15.55
    File "/home/jensens/workspace/pro/akifair/parts/instance/etc/package-includes/002-kup.akivdb-configure.zcml", line 1.0-1.54
    File "/home/jensens/workspace/pro/akifair/src/kup.akivdb/src/kup/akivdb/configure.zcml", line 8.2-8.37
    File "/home/jensens/.buildout/shared-eggs/collective.fingerpointing-1.8-py2.7.egg/collective/fingerpointing/configure.zcml", line 23.2-23.32
    AttributeError: 'module' object has no attribute 'handlers'
```

**Comment by hvelarde** [comment]
Python 3 compatibility is being addressed in #98.
